src/graphapi/opgraph.cpp

```diff
@@ -292,26 +279,31 @@ bool isIsomorphic(const OpGraph& left, const OpGraph& right)
 {
     if(left.numNodes() != right.numNodes())
     {
+        MIOPEN_LOG_I("test failed due to num nodes being different");
```

Comment:
[R] (in many places of this PR) I would use `MIOPEN_LOG_I2`, it is intended to log detailed info.